src/main/java/online/bookstore/dto/cart/AddCartItemDto.java

```diff
@@ -0,0 +1,13 @@
+package online.bookstore.dto.cart;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@Data
+public class AddCartItemDto {
+    @NotBlank
```

Comment:
Use `@NotBlank` for String

```suggestion
    @NotNull
    @Positive
```